src/sqlparser/src/ast/mod.rs

```diff
@@ -1491,6 +1491,21 @@ pub enum Statement {
         param: Ident,
         value: SetVariableValue,
     },
+    /// DECLARE CURSOR
+    DeclareCursor {
+        /// View name
+        cursor_name: ObjectName,
+        query: Box<Query>,
+    },
+    /// FETCH FROM CURSOR
+    FetchCursor {
+        cursor_name: ObjectName,
+        count: Option<i32>,
```

Comment:
Can we add some documentation indicating what `Option` is for here?

Reply:
done